sympy/printing/str.py

```diff
@@ -44,6 +44,16 @@ def emptyPrinter(self, expr):
         else:
             return str(expr)
 
+    def _print_Reals(self,expr):
+        return "S.Reals"
+
+    def _print_Complexes(self,expr):
+        return "S.Complexes"
+
+    def _print_Naturals(self,expr):
```

Comment:
The methods in this file are (mostly) in alphabetical order. You should add these in accordingly.